R/verb-arrange.R

```diff
@@ -45,7 +50,15 @@ op_desc.op_arrange <- function(x, ...) {
 #' @export
 sql_build.op_arrange <- function(op, con, ...) {
   order_vars <- translate_sql_(op$dots, con, context = list(clause = "ORDER"))
-  group_vars <- c.sql(ident(op_grps(op$x)), con = con)
-
-  select_query(sql_build(op$x, con), order_by = order_vars)
+  if(op$args$.by_group) {
```

Comment:
Needs space after `if`

Reply:
I think it would be slightly nicer to create a new `order_by` variable inside the if statement, and then do `select_query(sql_build(ops$x, con), order_by = order_by)`